cmd/create/machinepool/nodepool.go

```diff
@@ -394,7 +394,8 @@ func addNodePool(cmd *cobra.Command, clusterKey string, cluster *cmv1.Cluster, r
 		}
 	} else {
 		r.Reporter.Infof("Machine pool '%s' created successfully on hosted cluster '%s'", createdNodePool.ID(), clusterKey)
-		r.Reporter.Infof("To view the machine pool details, run 'rosa describe machinepool --machinepool %s'", name)
+		r.Reporter.Infof("To view the machine pool details, run 'rosa describe machinepool  -c %s --machinepool %s'",
```

Comment:
NP : remove the extra space :pray: 
```suggestion
		r.Reporter.Infof("To view the machine pool details, run 'rosa describe machinepool -c %s --machinepool %s'",
```